acc/components/optics/guide_baseline.py

```diff
@@ -189,6 +189,10 @@ def __init__(
             float(ww), float(hh), float(hw1), float(hh1), float(l),
             float(R0), float(Qc), float(alpha), float(m), float(W),
         )
+        import mcni
+        neutrons = mcni.neutron_buffer(1)
+        neutrons[0] = mcni.neutron(r=(0,0,0), v=(0,0,1000), prob=1, time=0)
```

Comment:
May need to be positioned so as to hit something on the way through so as to trigger compilation of `calc_reflectivity`?

Reply:
good idea!

Reply:
Sorry I only saw this after merging the PR. Please feel free to improve it